### ISSUE TITLE
As a User I want to see as much as possible Rich Text editor

### ISSUE THREAD

**Issue by kalitine** [coast-team/mute#41]
Find below a list of features about Rich Text Editor.
This list is based on this [Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet). If there are leftovers, do not hesitate to notice it !

Most of those features work thanks to [HyperMD](https://github.com/menou2u/HyperMD).

If there's no particular mention to it, the feature runs on both Chrome and Firefox.

## Inline rendered style Markdown ( #26 )
- [x] Headers (from H1 to H6)
- [x] Look alike H1 via = line
- [x] Look alike H2 via - line
- [x] Integration of code via ```
- [x] Bold style (** and __)
- [x] Italic style (* and _)
- [x] Both bold and italic style (sometimes the syntax does not show up well when it is focused)
- [x] Crossed style via ~~
- [x] Lists (1., *, - and +)
- [ ] Paragraph within lists via ⋅⋅⋅
- [ ] Tables
- [x] Quotations via > (sometimes if there's syntax inside it does not show up well when it is focused)
- [ ] Inline HTML (balises are highlighted but not interpreted)
- [x] Horizontal rule (---, *** or ___ lines)
- [x] Links
- [x] Integrate shortcuts ( #21 )

## Handling images ( #43 )
- [x] Load images from an URL with a reference
- [x] Load images from an URL without a reference
- [x] Load images from clipboard

## Math formulas rendering ( #28 )
This part works thanks to MathJax, a lib included in HyperMD.
- [x] Load Math formulas with a preview rendering
- [x] Load Math formulas without a preview rendering
- [ ] Create an option so the user can choose to have or not a preview
- [ ] List all the [configs](http://docs.mathjax.org/en/latest/config-files.html#common-configurations) available for MathJax and find the most suitable one

## Opened issues of HyperMD that impact Rich Text Editor
HyperMD's issues are in Mandarin. But there's a [Changelog.md](https://github.com/laobubu/HyperMD/blob/master/CHANGELOG.md) file which references the features HyperMD's owner expect to have in future releases.
* Tables aren't rendered
- [ ] add: Table Parsing
- [ ] add: Table Styling

* Basically our #20 (edit: closed)
~~- [ ] toolbar styling buttons~~

* A feature for Math rendering
- [ ] side-by-side preview (with div-to-line mapping support)

* A possible feature for table
- [ ] add: Table Mode Keyboard Binding
